schedule/schedule-appengine/src/main/i18n/locale-en.json

```diff
@@ -213,5 +215,17 @@
   "VALUE_MUST_BE_LESS_THAN": "Value must be less than {{value}}",
   "VIEW_SUBSCRIBERS": "View Subscribers",
   "WEDNESDAY_ABBREVIATED": "Wed",
-  "WIDGETS": "Widgets"
+  "WIDGETS": "Widgets",
+  "HISTORY_TYPE": "History event type",
+  "HT_ALL": "Show all events",
```

Comment:
That I saw but the page is only visible to admins so wasnt sure if you also wanted to eventually show the page to clients. For clients we would probably only want to show a subset of events, for example failed to send email to jasify owners should only be visible to admins. For the admin only events I assume we would not do translations right?
By the way the file was alphabeticaly sorted to make looking up of exisitng translations easier. Some of the translations you have added already exist under a different key, eg HT_Logout vs SIGN_OUT.